src/resmom/linux/alps.c

```diff
@@ -205,13 +234,27 @@ typedef struct ud {
 	int stack[MAX_BASIL_STACK + 1];
 	char status[BASIL_STRING_SHORT];
 	char message[BASIL_ERROR_BUFFER_SIZE];
+	char type[BASIL_STRING_SHORT];
+	char basil_ver[BASIL_STRING_SHORT];
 	char error_class[BASIL_STRING_SHORT];
 	char error_source[BASIL_STRING_SHORT];
 	element_counts_t count;
+	element_counts_sys_t count_sys;
 	inventory_data_t current;
+	system_data_t current_sys;
 	basil_response_t *brp;
 } ud_t;
 
+/**
+ * Pointer to a response structure (that gets filled in with KNL Node information).
+ */
+static basil_response_t *brp_knl;
```

Comment:
Done , moved it to create_vnodes_knl

Reply:
brp_knl is specific to alps. so don't think we should add it to dep_topology.  it's been used in multiple function so kept it as global for convenience. If it's ok i would like to keep it as it is, otherwise i might have to make it local and move around by changin few function definition.